App/src/services/dossier.ts

```diff
@@ -42,3 +43,15 @@ export function getDossierRequiredReview(): Promise<GetMyDossiersResponse> {
 export function getDossierReport(id: string): Promise<DossierReportResponse> {
     return axios.get(`/Dossier/GetDossierReportByDossierId/${id}`);
 }
+
+export function searchDossiersByTitle(title: string): Promise<GetMyDossiersResponse> {
```

Comment:
There should only be 1 endpoint, not 3. The user should be able to enter their parameters and click search, and the system searches for all the parameters (eg `/Dossier/SearchDossiers?title=some name&state=0&groupId=someId`)